src/main/scala/com/itechart/ny_accidents/constants/GeneralConstants.scala

```diff
@@ -38,4 +38,9 @@ object GeneralConstants {
   val HTML_ELEMENT_DAY_SELECTOR = ".day"
   val HTML_ELEMENT_DATE_ATTRIBUTE = "rel"
   val HTML_ELEMENT_TD = "td"
+
+  val HASH_DIFFERENCE: Long = MILLIS_IN_HOUR
+  val FIRST_STATION_ID = 1
```

Comment:
What the magic number? Why it 1 and 3, but not 7 and 13 for instance?

Reply:
Yes, we always will use station id's from 1 to 3. Cause we have only 3 weather stations in New York

Reply:
Please, add some comments in code about that

Reply:
Done